colin/core/result.py

```diff
@@ -97,6 +99,44 @@ def save_json_to_file(self, file):
                   fp=file,
                   indent=4)
 
+    @property
+    def xunit(self):
+        """
+        Get the xunit representation of results
+
+        :return: str
+        """
+
+        top = Element("testsuites")
+
+        testsuite = SubElement(top, "testsuite")
+        top.append(testsuite)
```

Comment:
`SubElement(top, "testsuite")` already added child element to the top and calling append
will cause two the same elements in the `<testsuites>`